src/core/variable.jl

```diff
@@ -133,3 +133,69 @@ function variable_reactive_bus_power_slack(pm::GenericPowerModel; nw::Int=pm.cnw
         start = PMs.getval(ref(pm, nw, :bus, i), "q_slack_start", cnd)
     )
 end
+
+"generates variables for both `active` and `reactive` power flow at each transformer"
+function variable_tp_trans_flow(pm::GenericPowerModel; kwargs...)
+    variable_tp_trans_active_flow(pm; kwargs...)
+    variable_tp_trans_reactive_flow(pm; kwargs...)
+end
+function variable_tp_trans_active_flow(pm::GenericPowerModel; nw::Int=pm.cnw, bounded=true)
```

Comment:
I have been trying to roughly enforce python pep8 conventions in TPPMs especially for function declarations, i.e. two lines between functions, and a docstring for each function. This just helps me with readability and easier debugging later. Docstrings don't all need to be complicated, just enough that I can easily understand the purpose of the function. Please check other functions in this file and others as well (also I noticed this issue in opendss.jl), I didn't mark each one.

Reply:
If some of the explanations are still unclear, please let me know.

Reply:
Thanks for that. I did see a couple of functions that you missed: `adjust_base_rec!` and `add_shunt!` in `opendss.jl`